### PR TITLE
docs(README): update README

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#308 

## Details
- remove outlayer.
- synchronize documents.

plz, check [this document](https://github.com/naver/egjs/wiki/Download-and-Using-egjs). 

## Preferred reviewers
@naver/egjs-dev

### CODE REVIEW

**Comment by netil** [comment]
LGTM

File: README-ko.md [hunk 1]
```diff
@@ -66,9 +84,9 @@ $ bower install egjs
 ## 의존성
 egjs는 다음의 라이브러리들에 대한 의존성을 가지고 있다.
 
-|[jQuery](https://jquery.com/)|[Hammer.js](http://hammerjs.github.io/)|[Outlayer](https://github.com/metafizzy/outlayer/)|
-|---|---|---|---|
-|1.7.0+ |2.0.4+|1.4.2+|
+|[jQuery](https://jquery.com/)|[Hammer.js](http://hammerjs.github.io/)|
```
Comment (mixed) [inline README-ko.md]: I think that Document need some description for before version.
`If you use egjs 1.3.0- that you need to outlayer.`

Comment (sculove) [inline README-ko.md]: @mixed if user use 1.3.0- version, we supports README of tag like this.
https://github.com/naver/egjs/tree/1.2.0

Comment (netil) [inline README-ko.md]: I agreen with @sculove. It can't be clarified all histories in readme file.

**Pull request merged**